omod-2.0/pom.xml

```diff
@@ -110,7 +110,7 @@
 
     <dependency>
       <groupId>org.openmrs.module</groupId>
-      <artifactId>webservices.rest-omod</artifactId>
```

Comment:
Did you change this accidentally? If not, what was the reason behind the change?

Reply:
Resolved thanks